5d745e799a72d600095eb7af/cmd

```diff
@@ -0,0 +1 @@
+%<a-s><a-k>mo<ret>d/bl<ret>Ec<c-n><esc>/or<space><ret>Na<backspace><ret><space><esc>l<gt><a-i>bs=+<ret>c===<esc>,q
```

Comment:
1. I'd replace `%<a-s><a-k>mo<ret>d` with `%smo<ret>xd`: no gain, but easier to type.
2. The last part starting with `/or<space><ret>` could be simplified to
```
/<space>r<ret>Ni<ret><esc><gt><a-C><a-w>ec===<esc>,q
```